allel/stats/roh.py

```diff
@@ -150,6 +149,116 @@ def _mhmm_predict_roh_state(model, is_het, pos, is_accessible, contig_size):
     return predictions, observations
 
 
+def roh_poissonhmm(gv, pos, phet_roh=0.001, phet_nonroh=(0.0025, 0.01), transition=1e-3,
+                   window_size=1000, min_roh=0, is_accessible=None, contig_size=None):
+
+
+    """Call ROH (runs of homozygosity) in a single individual given a genotype vector.
```

Comment:
Usually docstring follows function declaration on next line. Would you mind running:
```
flake8 --max-line-length=100 allel/stats/roh.py
```
...and fixing any warnings or errors. (This is run under travis CI for python 3.6, but that build is currently broken by recent changes to array print formating in numpy, so I'm not expecting that build to pass.)